### ISSUE TITLE
Disable Admin Link

### ISSUE THREAD

**Issue by smth** [lektor/lektor#304]
It'd be useful to be able to prevent the addition of the admin link (pencil), via an option in the project file perhaps. If this is already possible, I can't find it mentioned in the docs.

**Comment by singingwolfboy** [comment]
I find this surprising. The admin is one of Lektor'a best features -- why do you want to disable that link? Can you explain your use-case?

**Comment by smth** [comment]
Agreed that admin is a nice feature, and I'm not talking about disabling "admin" just the link to it.

My use case is that I'm a visual designer, I often design in code, often running Lektor. The box in the top right of the page can be distracting and sometimes obscures an element that I am designing.

If I want to get to admin section, there are other simple ways to get there, such as typing `admin` in the address bar. So on balance, I'd rather the pencil icon wasn't there.

I could of course use CSS to move/hide it, but I'd prefer not to include Lektor specific code.

**Comment by seblu** [comment]
I have a similar issue, the link hides a menu item. I use an ad-blocker to hide it.

**Comment by torian257x** [comment]
@seblu just saying: you can add a little bit of css to remove the link or move it somewhere else, right? it's just another html element on that page

**Comment by seblu** [comment]
I did not think about it. The following code do the job, thanks.

```
#lektor-edit-link {
 visibility: hidden;
}
```
